### PR TITLE
Add setting to not swap in the specified glob

### PR DESCRIPTION
**Pull request by mrasu**

### CODE REVIEW

File: lib/wovnrb/services/glob.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module Wovnrb
+  # inspired from https://github.com/isaacs/node-glob
```
Comment (jsand-minimal) [inline lib/wovnrb/services/glob.rb]: inspired by

**Comment by jsand-minimal** [comment]
How do I say I want to ignore everything under the api path, but I want to include api?

LOCALIZE
/api

IGNORE
/api/hi
/api/yo
/api/anything

**Comment by mrasu** [comment]
@jsand-minimal 
you can configure
`api/*` for directly, 
`api/**/*` for including subdirectly

**Comment by jsand-minimal** [comment]
ok

**Pull request merged**